client/config.go

```diff
@@ -78,8 +77,11 @@ type Configuration struct {
 	// time to use when sleeping between a failed health check and the next check.
 	BackgroundHealthCheckFailThrottleFactor float64 `yaml:"backgroundHealthCheckFailThrottleFactor" validate:"min=0,max=10"`
 
-	// HashingConfiguration is the configuration for hashing of IDs to shards
+	// HashingConfiguration is the configuration for hashing of IDs to shards.
 	HashingConfiguration HashingConfiguration `yaml:"hashing"`
+
+	// The host and port on which m3db listens.
+	ListenAddress string `yaml:"listenAddress"`
```

Comment:
Did you need to add this?

Reply:
Yes, because `NewTopologyInitializer()` for the static config needs a listen address